stations.csv

```diff
@@ -3202,6 +3202,8 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 4194;Biot;biot;8775769;87757690;43.610951;7.126090;;FR;Europe/Paris;f;f;f;t;f;f;FRIBK;BJO;t;;f;8700723;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4194;
 4195;Grasse;grasse;8775772;87757724;43.653317;6.925524;;FR;Europe/Paris;f;f;f;t;f;f;FRIBL;GRA;t;;f;8705486;f;;f;;f;;f;;f;;f;8705486;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;グラース;그라스;;;Грас;;;格拉斯;urn:trainline:public:nloc:csv4195;
 4196;Ranguin;ranguin;8775773;87757732;43.5691636;6.9693134;;FR;Europe/Paris;f;f;f;t;f;f;FRIBM;RGN;t;;f;8704345;f;;f;;f;;f;;f;;f;8704345;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4196;
+4222;Casamarciano;casamarciano;;;40.93125823842119;14.548592083542841;;IT;Europe/Paris;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4222;
```

Comment:
Never reuse ids. Who knows what this 4222 represented in the past. New stations must be added at the end of the file.

Reply:
Also, which station is this? It doesn't have any vendor code and there seems to be no train station passing close.
![openstreetmap](https://github.com/trainline-eu/stations/assets/4478372/7e4fc535-9535-466b-9b66-85c5cfa9f28a)

https://www.openstreetmap.org/#map=15/40.9313/14.5486&layers=T

Reply:
Looks like it's a bus station
https://maps.app.goo.gl/gE6tHU52M3uDM9347

Reply:
Not sure what vendor it should use tho